core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/imagecatalog/ImagesToImagesV4ResponseConverter.java

```diff
@@ -89,6 +89,7 @@ public ImagesV4Response convert(Images source) {
     private void copyImageFieldsToJson(Image source, ImageV4Response json) {
         json.setDate(source.getDate());
         json.setCreated(source.getCreated());
+        json.setPublished(source.getPublished());
```

Comment:
what happens if published field is empty in image catalog? will it cause any issue? or will we fill in this field for every image in the catalog?

Reply:
@horadla23 no, it won't cause any issue. On API level it is nullable. But we are going to fill them out manually at least for prod catalog (CB-15616) so it won't be empty or null or whatever on the UI (CB-15610).